### DISCUSSION TITLE
Update-2022-11-04

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#166] [Announcements]
Implemented a first pass at user interaction. Maybe the beginnings of a game engine? Who knows.

Added `interactor` class which handles keyboard events. Demo in `demo-interactor.lisp`.

https://user-images.githubusercontent.com/103860793/200096424-1b863f64-65e5-420b-94de-e9e2e5275785.mov

**Comment by ghost** [comment]


**Reply by kaveh808** [reply]
I plan on adding collisions as I develop the interaction tools.
